lib/fake-server/index.js

```diff
@@ -251,7 +251,7 @@ var fakeServer = {
             }
             if (/\*/.test(url)) {
                 // eslint-disable-next-line no-param-reassign
-                url = url.replace(/\/\*/g, "/(.*)");
+                url = url.replace(/\/\*/g, "/*path");
```

Comment:
I am not sure I follow this. Could you add a small comment inline here on what is happening here? I assume the `/*path` bit is some `path-to-regexp` syntax, but I failed in finding/understanding which part of the docs apply. This was the closest fit I could find: https://github.com/pillarjs/path-to-regexp#unexpected--or-

Related: https://sinonjs.github.io/nise/#serverrespondwithurlregexp-response

Reply:
Ah, not sure how I missed this being explained in the description 🙈

Reply:
No worries, thanks for the sanity check 😁